src/components/CustomerDetails/InputPassportSection.tsx

```diff
@@ -61,6 +63,13 @@ export const InputPassportSection: FunctionComponent<InputPassportSection> = ({
       : setIdInput("");
   }, [selectedCountry, trimmedPassportNum, setIdInput]);
 
+  useEffect(() => {
```

Comment:
possible to leave some comments on why this is required/what it does?

Reply:
added the comment.